### ISSUE TITLE
Pockettube on Firefox spams 'sign-in with google' page

### ISSUE THREAD

**Issue by JacobTravers** [NabokD/pockettube#37]
I am already signed in with google so I am not sure why pockettube spams me with up to 5 new tabs opening the 'sign in with google' page, if I go through with the sign-in attempt it asks me to give new permissions for youtube and google drive. If I do so and continue it says there was an error.

I am on mozilla firefox 107.ob8 I have tested it after disabling all other extensions, this extension seems to be the issue

**Comment by NabokD** [comment]
I can't reproduce it. Try to logout in the extension popup

**Comment by JacobTravers** [comment]
I believe this may have been an issue with the version of Firefox I was running. I've updated and the issue is gone so I will close this. Thank you for the quick response!

**Comment by ieremies** [comment]
I don't think that was a Firefox version issue. I was running the latest (107.0.1) and still experiencing the issue. Every time I opened the subscriptions page, it would ask me to login, no matter how many times. But seeming that @JacobTravers got (somehow) it working, I tried to uninstall all other extensions, did the login, and it stopped, even after I re-enabled all extensions.

For future reference, I was running Enhancer for YouTube and Unhook (and others non-youtube related). If had to guess, Unhook's feature to redirect to the subscription page every time you land in the home page was the problem, but it is only a guess.
